lesson_5/lesson_5.py

```diff
@@ -0,0 +1,45 @@
+# coded: UTF-8
+
+# 1 Task
+
+import re
+
+input_value = input('Зробіть введення значення: ')
+
+string_regex = "^[a-zA-Zа-яА-ЯёЁЄєІіЇї]+$"
+symbol_regex = "^[&^_@#\-!$%^&*()£€¥§°¢_+|~=`{}\[\]:\";'<>?,.\/\\\]+$"
+number_regex = "^\d+$"
+
+string_pattern = re.compile(string_regex)
+symbol_pattern = re.compile(symbol_regex)
+number_pattern = re.compile(number_regex)
+
+if len(input_value) == 1:
+    if number_pattern.search(input_value):
```

Comment:
there is a better way to check if a character or a string is a digit: str.isnumeric() or str.isdigit(). Why did you decide to use regular expressions here?